packages/api/src/utils/billing.js

```diff
@@ -1,24 +1,40 @@
 /* eslint-disable no-void */
 
+import { InvalidTosAgreementError, TosAgreementRequiredError } from '../../src/errors.js'
+
 /**
  * @typedef {import('./billing-types').StoragePriceName} StoragePriceName
  */
 
+/**
+ * @param {import('./billing-types').Agreement} agreement
+ * @returns {boolean}
```

Comment:
It might be nice to type this as a type guard for tsc. https://goulet.dev/posts/type-guard-in-jsdoc/

I'm not positive this exact suggestion is the right syntax with the `import`, but might work. Otherwise you can use a typedef to alias agreement into this file like

```
/**
 * @typedef {import('./billing-types').Agreement} Agreement
 */
```

and then do 
```
 * @returns {agreement is Agreement}
```

```suggestion
 * @returns {agreement is import('./billing-types').Agreement}
```

If that's not aligned with the point of this function, ignore

Reply:
updating